modules/ROOT/pages/install-workflow.adoc

```diff
@@ -1,12 +1,79 @@
-= Installation Workflow
+= Installing Anypoint Platform Private Cloud Edition (Anypoint Platform PCE)
+:page-aliases: prereq-gravity-check.adoc
 
-== Install Anypoint Platform Private Cloud Edition (Anypoint Platform PCE)
+The process to install Anypoint Platform Private Cloud Edition (Anypoint Platform PCE), consist of the following steps:
 
-To install Anypoint Platform PCE, MuleSoft professional services must be involved throughout the installation process. Prior to installation, have your infrastructure team verify that the required infrastructure is correctly installed and configured as described in xref:install-checklist.adoc[Product Prerequisites]. 
+. Setting up and verifying your infrastructure
+. Performing a pre-installation verification
+. Installing Anypoint Platform PCE
 
-== Install Anypoint Monitoring and Anypoint Visualizer
+[NOTE]
+You must contact MuleSoft Professional Services prior performing the pre-installation verification and installation of Anypoint Platform PCE, as they are involved in these processes. 
```

Comment:
```suggestion
Contact MuleSoft Professional Services before performing the pre-installation verification and installation of Anypoint Platform PCE because they are involved in these processes. 
```